README.markdown

```diff
@@ -464,7 +464,11 @@ manifests as a valid password attribute. *Type*: rvalue
 * `to_bytes`: Converts the argument into bytes, for example 4 kB becomes 4096.
 Takes a single string value as an argument. *Type*: rvalue
 
-* `type`: Returns the type when passed a variable. Type can be a string, array, hash, float, integer, or boolean. *Type*: rvalue
+* `type3x`: Returns a string description of the type when passed a variable. Type can be a string, array, hash, float, integer, or boolean. This function will be removed when puppet 3 support is dropped and the new type system may be used. *Type*: rvalue
+
+* `type_of`: Returns the literal type when passed a variable. Requires the new
+  parser. The output is meant for human use; for programatic inspection of types
```

Comment:
I think I like the look of `=~ Numeric` over `<= Numeric`. And yeah, I see your point, now. I wrote that before I knew what it was for.